File: 백준/레벨Bronze V/2753. 윤년/윤년.ts
import fs = require('fs');

const input: string = fs.readFileSync(0, 'utf-8').trim();
const year: number = parseInt(input);

const result = (year % 4 === 0 && year % 100 != 0) || year % 400 === 0;

console.log(result ? 1 : 0);

File: 백준/레벨Bronze V/2753. 윤년/윤년.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());

        int year = Integer.parseInt(st.nextToken());

        boolean result = (year % 4 == 0 && year % 100 != 0) || year % 400 == 0;

        System.out.println(result ? 1 : 0);
    }
}